src/components/articles/AllArticlesCard.jsx

```diff
@@ -0,0 +1,17 @@
+import "./AllArticles.css";
+
+export default function AllArticlesCard({ article }) {
+  return (
+    <li className="all-articles-card">
+      <h2 id="all-articles-title">{article.title}</h2>
+      <h3 id="all-articles-topic">{article.topic}</h3>
+      <p id="all-articles-author">{article.author}</p>
+      <img id="all-articles-img" src={article.article_img_url} />
+      <div id="all-articles-votes-comments">
+        <div>♥️ {article.votes}</div>
+        <div>💬 {article.comment_count}</div>
+      </div>
```

Comment:
I love the use of symbols/emojis here to eliminate the need for labels but still communicate information
For a more professional look, consider using something like React icons
https://react-icons.github.io/react-icons/